<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SCTB404" ignored="0" name="Test" tests="1" failures="1" timestamp="10 Jun 2020 14:34:36 GMT" time="187.117" errors="0">
  <testcase name="feature" time="186.46" classname="cucumber.api.testng.AbstractTestNGCucumberTests">
    <failure type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: expected [true] but found [false]">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: expected [true] but found [false]
at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
Caused by: java.lang.AssertionError: expected [true] but found [false]
at com.qualiprobdd.e2etests.Fournisseur.models.EvalFourModel.Vérifier_historique(EvalFourModel.java:392)
at com.qualiprobdd.e2etests.Fournisseur.stepdefs.EvalFourStepDef.Vérifier_historique_des_évaluations(EvalFourStepDef.java:120)
at ✽.And Vérifier historique des évaluations(src/specs/features/Fournisseur/EvaluationFournisseur.feature:26)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- Test -->
